app/jobs/stockit_delete_job.rb

```diff
@@ -1,13 +1,17 @@
 class StockitDeleteJob < ActiveJob::Base
   queue_as :stockit_updates
 
-  def perform(inventory_number)
-    response = Stockit::Browse.new(inventory_number).remove_item
+  def perform(package_id)
+    package = Package.find_by(id: package_id)
```

Comment:
ah good catch! I had replaced inventory_number with the package object in Stockit::Item so that everything was consistent but I understand the problem you mean. Will have a think and update the PR.